2017-04/april-08.md

```diff
@@ -0,0 +1,83 @@
+## April 8 ([discuss](https://github.com/babel/notes/pull/19))
+
+### Babel 7 Updates
+
+- [Upgrade to Babel 7](https://github.com/babel/babel.github.io/pull/1146): We posted this for the alpha version already (v7.0.0-alpha.7), which we don't make any guarantees on (although we are using it at Behance).
+
+### Getting Contributors
+
+- [Rails Girls Summer of Code](https://teams.railsgirlssummerofcode.org/projects/177-babel) submissions are over. Next on the timeline is waiting for acceptance letters on May 1.
+- [Google Summer of Code](https://summerofcode.withgoogle.com/organizations/5842528113786880/): We're reviewing applications at the moment.
+- How can we reach out to companies that are using Babel for contributions/maintaince help? Had some fun with a small little April Fools joke that sparked some discussion about OSS and funding. We'll be talking to some interested companies/teams that want to contribute dev time to the project (roadmap below).
+- We're on [Open Collective](https://opencollective.com/babel) now! Better than funding, we'd encourage you to ask your employee to donate, or for them to donate dev time to the project to help us out.
+
+If you have any suggestions on how we can do more, please reach out! (comment here, join [Slack](https://slack.babeljs.io/), or [Twitter](https://twitter.com/babeljs).
```

Comment:
Probably don't need the parens around the second sentence. Maybe `If you have any suggestions on how we can do more, please reach out by commenting here, on [Slack](https://slack.babeljs.io/), or on [Twitter](https://twitter.com/babeljs).`?